docs/docs/APIdocs/layers/nn/Simple Layers/Clamp.md

```diff
@@ -0,0 +1,52 @@
+## Clamp ##
```

Comment:
Don't you read the document of the layer...? Obvious it's a non-linear activation?